schema/resource.go

```diff
@@ -91,6 +95,32 @@ func (r *Resource) Columns() []string {
 	return r.Table.Columns.Names()
 }
 
+func (r *Resource) CalculateUniqueValue() error {
+	names := r.Table.PrimaryKeys()
+	if len(names) == 0 {
+		names = r.Table.Columns.Names()
+	}
+	slices.Sort(names)
+	value := make([]any, len(names))
+	for i, name := range names {
+		value[i] = r.Get(name)
```

Comment:
It's also adding an empty `_cq_id` here I think. (also `_cq_parent_id` is still here, but I now think it should be included in the hash, so no worries there)

Reply:
But as long as it always adds the empty object it is fine